source/Mission.cpp

```diff
@@ -97,6 +97,8 @@ namespace {
 				return "on visit";
 			case Mission::Trigger::WAYPOINT:
 				return "on waypoint";
+			case Mission::Trigger::DAILY:
+				return "on daily";
```

Comment:
This feels weird to me, daily yes but on daily that's weird.
Idk how to fix it - or if we can , given the syntax is always "on ..." Maybe on day? Other suggestions would be great

Reply:
How about on every day?

Reply:
That sounds good to me actually, I don't think there is a hard rule that it needs to be in one word.

Reply:
done